### ISSUE TITLE
Screen reader virtual cursor reads controls

### ISSUE THREAD

**Issue by krigersergei** [WICG/inert#121]
Hi, I experience this issue with `inert`:

once `inert` is added it prevents controls from being focused, but it still possible to navigate to controls inside an element with `inert` with a screen reader virtual cursor and use them (see screenshot).

Is it planned to be so and if yes what can be a solution to **totally** prevent elements inside `inert` from being clicked? I only can think of adding `aria-hidden` for all the controls which doesn't seem a good idea.

![virtual-cursor](https://user-images.githubusercontent.com/10166916/58593980-b833eb80-826c-11e9-938b-408e9d75518b.gif)

**Comment by robdodson** [comment]
The inert polyfill should be adding aria-hidden to the container so any children should inherit that. If that's not working then it's a bug.

**Comment by robdodson** [comment]
This looks like it might be a Chrome bug. It works as expected in Safari.

**Comment by robdodson** [comment]
I filed a Chromium bug to track this:
https://bugs.chromium.org/p/chromium/issues/detail?id=971880

**Comment by krigersergei** [comment]
Hi @robdodson, thanks a lot! I hope it's gonna be fixed at some point.
